scripts/kind-dev-cluster

```diff
@@ -30,58 +33,86 @@ readonly SKUPPER_KUBE_ADAPTOR_IMAGE=${SKUPPER_KUBE_ADAPTOR_IMAGE:-${REGISTRY}/ku
 readonly SKUPPER_NETWORK_OBSERVER_IMAGE=${SKUPPER_NETWORK_OBSERVER_IMAGE:-${REGISTRY}/network-observer:${IMAGE_TAG}}
 
 KIND_LOG_LEVEL="1"
-DEBUG=${DEBUG:=false}
+VERBOSE=${VERBOSE:=false}
 CLUSTER="skupper-dev"
 IMAGE_SOURCE="docker"
 FORCE_RECREATE="false"
 METALLB="false"
 SUBNET="1"
 
-if [ "${DEBUG}" == "true" ]; then
-  set -x
-  KIND_LOG_LEVEL="6"
-fi
+verbose_log() {
+	if [ "${VERBOSE}" == "true" ]; then
+		echo "(skdev) $1"
+	fi
+}
 
 ensure::kind() {
 	if ! command -v "${KIND}" > /dev/null 2>&1; then
-		echo "${KIND} not found";
+		echo "(skdev) ${KIND} not found, exiting";
 		echo "See https://kind.sigs.k8s.io/ for installation and usage.";
 		exit 1
+	else 
+	   verbose_log "Found Kind ..."
 	fi
 }
-ensure::docker() {
-	if ! command -v "${DOCKER}" > /dev/null 2>&1; then
-		echo "${DOCKER} not found";
-		echo "This tool assumes docker is used to run kind and does not support the experimental podman provider";
+
+ensure::container() {
+	if [[ "$CONTAINER" == "docker" ]]; then
```

Comment:
CI Error here:
```
+ KUBECONFIG=/home/circleci/kc.east
+ ./scripts/kind-dev-cluster -n east --images ociarchive
./scripts/kind-dev-cluster: line 60: CONTAINER: unbound variable
```

Maybe we set a default CONTAINER=docker at the top of the script?

Reply:
> CI Error here:
> 
> ```
> + KUBECONFIG=/home/circleci/kc.east
> + ./scripts/kind-dev-cluster -n east --images ociarchive
> ./scripts/kind-dev-cluster: line 60: CONTAINER: unbound variable
> ```
> 
> Maybe we set a default CONTAINER=docker at the top of the script?

Done !! Thanks.